tooling/api/src/shell.ts

```diff
@@ -5,6 +5,24 @@
 import { invokeTauriCommand } from './helpers/tauri'
 import { transformCallback } from './tauri'
 
+interface SpawnOptions {
+  // Current working directory.
+  cwd?: string
+  // Environment variables. set to `null` to clear the process env.
+  env?: { [name: string]: string }
```

Comment:
we should change the comments to use `/** */`, otherwise it won't appear in the auto-generated docs
```suggestion
  /** Current working directory. */
  cwd?: string
  /** Environment variables. set to `null` to clear the process env. */
  env?: { [name: string]: string }
```

Reply:
I knew i was doing it the wrong way :D thanks!